### PR TITLE
script for updating local superdesk repository

### PR DESCRIPTION
**Pull request by mugurrus**
It updates your local SD repo with one single command:
- stashes your local changes
- pulls latest updates (with rebase)
- pushes latest updates to your git repo
- npm update && bower update
- updates backend requirements (pip install)
- restores your local changes

You can specify which branch to use with: -b 'branch'
-h for help

### CODE REVIEW

File: scripts/sd-update [hunk 1]
```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -e
+
+clear_files="server/Procfile server/app.py"
+use_branch='master'
+force=''
+quick=false
+mydir=`pwd`
```
Comment (actionless) [inline scripts/sd-update]: `$(readlink -e $(dirname "$0"))`

File: scripts/sd-update [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    case "$1" in
+    -h) show_help; exit 0 ;;
+    -f|--force) force='--force'; shift ;;
+    -q|--quick) quick=true; shift ;;
+    -b|--branch) use_branch=$2; shift ;;
+    *) shift; break ;;
+    esac
+done
+
+for f in $clear_files; do
```
Comment (actionless) [inline scripts/sd-update]: what is the purpose of this part?

Comment (mugurrus) [inline scripts/sd-update]: I modify some files from the backend while working but I don't push these changes as they are not welcomed it seems. E.g. in app.py I always set debug = False (line 122). I don't want these changes to be overwritten, that's why I save these files in a temp dir.

File: scripts/sd-update (hunk 1 above)
Comment (mugurrus) [inline scripts/sd-update]: done

**Comment by actionless** [comment]
LGTM

**Pull request merged**